src/main/java/com/hotels/hcommon/hive/metastore/client/closeable/CloseableMetaStoreClientFactory.java

```diff
@@ -27,32 +26,36 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import com.hotels.hcommon.hive.metastore.exception.MetaStoreClientException;
 import com.hotels.hcommon.hive.metastore.client.api.CloseableMetaStoreClient;
 import com.hotels.hcommon.hive.metastore.client.api.MetaStoreClientFactory;
+import com.hotels.hcommon.hive.metastore.exception.MetaStoreClientException;
 
 public class CloseableMetaStoreClientFactory implements MetaStoreClientFactory {
   private static final Logger log = LoggerFactory.getLogger(CloseableMetaStoreClientFactory.class);
 
+  private static class DisabledHiveMetaHookLoader implements HiveMetaHookLoader {
+    @Override
+    public HiveMetaHook getHook(Table tbl) throws MetaException {
+      return null;
+    }
+  }
+
```

Comment:
This was refactored as a result of this type of warning: [SIC_INNER_SHOULD_BE_STATIC_ANON](http://findbugs.sourceforge.net/bugDescriptions.html#SIC_INNER_SHOULD_BE_STATIC_ANON)